### DISCUSSION TITLE
Upcoming Event：Ask Milvus Anything #1 at 20:00-21:00 (EDT) on Aug.25th

### DISCUSSION THREAD

**Discussion by kateshaowanjou** [milvus-io/milvus#3387] [Show and tell]
🗣️ Do you have any questions you want to ask Milvus core developers? Or do you have any ideas/experiences you want to share with other users?
💡 You are welcome to join this community event at 17:00-18:00 (PDT)/20:00-21:00 (EDT) on Aug.25th via Zoom.

🗓️ Register the event here: https://lnkd.in/gBP47DF

**Comment by gujun720** [comment]
On the meeting, we shared a quick look at how Milvus v0.11.0 manages data

https://www.youtube.com/watch?v=x2EketSWQLw
